includes/polling/os/nexans.inc.php

```diff
@@ -0,0 +1,4 @@
+<?php
+
+$version = snmp_get($device, 'infoMgmtFirmwareVersion.0', '-OQv', 'NEXANS-BM-MIB');
```

Comment:
Could you convert these two in to a multi get to save an snmp call please?

Reply:
Travis is checking it.